confluent_kafka/src/Consumer.c

```diff
@@ -550,17 +550,24 @@ static PyObject *Consumer_poll (Handle *self, PyObject *args,
 
 
 static PyObject *Consumer_close (Handle *self, PyObject *ignore) {
-	CallState cs;
```

Comment:
To be compatible with older compilers the variable definition needs to be at the beginning of the scope, so keep this line here.

Reply:
fixed